scripts/apps/monitoring/directives/MonitoringGroup.ts

```diff
@@ -141,14 +141,24 @@ export function MonitoringGroup(
                 );
             }
 
-            function scheduleQueryFn(event, data) {
-                queryItems(event, data, {auto: (data && data.force) ? 0 : 1})
+            function scheduleQueryFn(event?, data?) {
+                return queryItems(event, data, {auto: (data && data.force) ? 0 : 1})
                     .finally(() => {
                         scope.$applyAsync();
                     });
             }
 
-            const scheduleQuery = throttle(scheduleQueryFn, 1000);
+            let queryPromise = null;
+            const scheduleQuery = debounce((event, data) => {
```

Comment:
It would be good to add a comment here, explaining how is it different from debouncing only.